docs/fundamentals/artifacts.md

```diff
@@ -1,193 +1,77 @@
 ---
-title: Artifacts & Models
+title: Model
```

Comment:
@pollfly Model -> Models?

Reply:
Should it be consistent with the "Task" title? or should it be "Models" because it's about working with multiple model classes?

Reply:
@pollfly Let's go with plural: "Projects", "Tasks" and "Models"